platform/admin/modules/admin-modules-docs/pom.xml

```diff
@@ -59,9 +59,6 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>
-                            spark-core
-                        </Embed-Dependency>
                         <Import-Package>
                             !org.eclipse.jetty.websocket.*,
                             !sun.reflect,
```

Comment:
✏️ I don't think we need this line now since the spark-core isn't being embedded.